packages/project-disaster-trail/src/components/Game/OrbManager.js

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable import/no-named-as-default */
+import React, { useEffect, useState, memo } from "react";
+import { map } from "lodash";
+
+import Orb from "./Orb";
+
+const OrbManager = ({ count = 10 }) => {
+  let tempModels = [];
+  let tick = 0;
+  // how big is the circle the orbs 'float' up and down in
+  const period = 2;
+
+  for (let i = 0, model; i < count; i++) {
+    model = {};
+    model.x = Math.random() * window.innerWidth;
+    model.y = Math.random() * window.innerHeight;
+    model.velocity = {
+      x: (Math.random() - 2) * 10,
+      y: 0 // (Math.random() - 1) * 10
+    };
+
+    tempModels.push(model);
+  }
+
+  const [models, setModels] = useState(tempModels);
+
+  useEffect(() => {
+    let animationFrameId;
+
+    const animate = () => {
+      tempModels = [];
+      for (let i = 0, model; i < count; i++) {
```

Comment:
Thinking of future devs (which may well be us in months or devs who've never done animation), it would be nice to have a note above the sections within this for loop, even something as simple as `//move onto screen` (for the if statements below) or `// orb bounce` for the model.y